configuration/php/add-payment-methods.php

```diff
@@ -0,0 +1,29 @@
+<?php
+// The sandbox API key used in this example is public.
+// Don't submit any personally identifiable information in any requests made with this key.
+// Sign in to developer.dojo.tech to create your own private sandbox key and use that instead
+// for secure testing.
+
+namespace Test;
+require_once "vendor/autoload.php";
+
+use Dojo_PHP\ApiFactory;
+use Dojo_PHP\Model\CreatePaymentIntentRequest;
+use Dojo_PHP\Model\Money;
+use Dojo_PHP\Model\PaymentMethod;
+
+$apiKey = "sk_sandbox_c8oLGaI__msxsXbpBDpdtwJEz_eIhfQoKHmedqgZPCdBx59zpKZLSk8OPLT0cZolbeuYJSBvzDVVsYvtpo5RkQ";
```

Comment:
are we OK sharing real api keys? 
I assume most of merchant will be using this key in their test integration. When we apply rate limiting, it can cause issues.

Reply:
Yes, good points. That what I mentioned on the rate limiting session:
* We should take care about test keys
* This is a `public sandbox test key` which has many limits, compare to the real sandbox keys